eZ/Bundle/PlatformInstallerBundle/src/Resources/config/database.yml

```diff
@@ -0,0 +1,18 @@
+services:
+    ezplatform.installer.database.connection.factory:
+        class: EzSystems\PlatformInstallerBundle\Database\ConnectionFactory
+        arguments: ["@ezpublish.persistence.connection"]
+
+    ezplatform.installer.database.connection:
+        class: Doctrine\DBAL\Connection
+        factory: ["@ezplatform.installer.database.connection.factory", createConnection]
+
+    ezplatform.installer.database.platform.mysql:
```

Comment:
Why did you include a ".platform" suffix ? I'm not sure what it is meant as a reference to.